app/assets/stylesheets/darkswarm/_shop-navigation.css.scss

```diff
@@ -69,23 +71,26 @@ ordercycle {
 
     select,
     p {
-      width: inherit;
       display: inline-block;
       color: $white;
       background-color: transparent;
       border: 0;
-      margin-bottom: 0;
+      border-radius: 0 0.25em 0.25em 0;
       font-size: 1em;
       line-height: 1.3em;
+      margin-bottom: 0;
       padding: 0.5em 1.25em 0.5em 0.75em;
       height: 2.35em;
-      background-size: 30px auto;
-      border-radius: 0 0.25em 0.25em 0;
       min-width: 13em;
+      width: 200px;
 
       @media all and (max-width: 480px) {
         width: 100%;
       }
+
+      @media all and (min-width: 640px) and (max-width: 1024px), (min-width: 1200px) {
```

Comment:
> - the width is currently 200px, in this PR we make it 250px for screens between 640px and 1024px and screens larger than 1200px

why not that width for those screens between 1024px and 1200px? if it fits in smaller screens why doesn't it for those a bit bigger? Unless there's something obvious I'm not seeing to me that should from 640px onwards like

```
@media all and (min-width: 640px)  {
  width: 250px;
}
```

Reply:
for screens smaller than 1024px the OC selector is an horizontal bar. above 1024px it moves to the top right where it doesnt have space for this until the screen gets to 1200px